### PR TITLE
Created Merge PR script

### PR DESCRIPTION
**Pull request by kingbar1990**

### CODE REVIEW

File: github/MergePullRequest.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+let axios = require('axios')
+
+module.exports = function (access_token, username, repo, id) {
+  axios.defaults.headers.common['Authorization'] = `token ${access_token}`;
```
Comment (kevinsimper) [inline github/MergePullRequest.js]: You should not change the defaults, just pass it to the request :)

**Comment by kingbar1990** [comment]
@kevinsimper , Please check new version.

**Comment by kevinsimper** [comment]
Looks good. 

Then it should also work in test.js with mock. 

It would have been easier if you made it work in test.js first as we talked about on the video call. Remember we talked about that?

**Comment by kingbar1990** [comment]
Yes, I remember. I have to add tests with mock into tests.js.

**Pull request closed without merging**